src/amber/cli/commands/generate.cr

```diff
@@ -6,7 +6,7 @@ module Amber::CLI
       class Options
         arg "type", desc: "scaffold, model, controller, migration, mailer, socket, channel, auth", required: true
         arg "name", desc: "name of resource", required: true
-        arg_array "fields", desc: "name:string body:text age:integer published:bool"
+        arg_array "fields", desc: "user:ref name:string body:text age:integer published:bool"
```

Comment:
should we be consistent and not use abbreviations `references`? See https://www.google.com/search?q=ref+abbreviation&oq=ref+abbr&aqs=chrome.0.0j69i57j0l4.3968j0j1&sourceid=chrome&ie=UTF-8

Also, also what do you think of something like `Post  title:string body:text belongs_to:user has_many:comments`

Reply:
This PR supports both `ref` and `reference`.  

Regarding `references`, This is describing the type of field so it should not be plural since its a noun, not an action. It answers the question of what type the field is.  `This field is a reference`, not `This field is a references`.

I will change the code to use the unabbreviated `reference` so it's clear.

Regarding the `belongs_to:user` and `has_many:comments`, we should discuss if we want to allow for mixing the order of the `name:type` and how we would do that.  I will leave that for a different PR.